### ISSUE TITLE
[feature request]Sort in random order

### ISSUE THREAD

**Issue by RanRanMiao** [SubhamTyagi/Last-Launcher#156]
Can the order of Apps be shuffled and not be arranged in any order?  It makes for a better look.

**Comment by Manmade290** [comment]
Yeah. I want to try this for some fresh feeling too, always watching the same order makes me bored.

**Comment by SubhamTyagi** [comment]
> Can the order of Apps be shuffled and not be arranged in any order? It makes for a better look.

And for worst usability 😀

**Comment by TIANGeng708** [comment]
I have submitted a solution for this issue.  This gives the user more choices and a better look.
